app/models/service_template.rb

```diff
@@ -450,6 +451,10 @@ def self.display_name(number = 1)
     n_('Service Catalog Item', 'Service Catalog Items', number)
   end
 
+  def my_zone
+    zone&.name if atomic?
```

Comment:
Why only `if atomic?` ? 

Service Bundles run automation which might need to be zone aware.

cc @tinaafitz

Reply:
Discussed this with @tinaafitz and Peter. We think each individual catalog items could specify their zones to run in, but the catalog bundle is zone-agnostic as it’s not a real workflow item, more of an organizational / container thing.

Reply:
Ok, I think that deserves a comment in the method otherwise it will be tough to recall why this decision was made.

Reply:
Looks like rubocop does not like `it's`

This should probably read:
```
# Catalog items can specify a zones to run in.
# Catalog bundle are used for grouping catalog items and are therefore treated as zone-agnostic.
```